carbon-design-system/carbon-addons-iot-react
# docs(table): add story and mdx for toolbar

Closes #3329

**Summary**
New story and updated docs for the toolbar

**Change List (commits, features, bugs, etc)**

- added new story
- updated table.main.story
- Added mdx
- smaller updates to table.story.helper

**Acceptance Test (how to verify the PR)**
- Check out story here https://deploy-preview-3366--carbon-addons-iot-react.netlify.app/?path=/story/1-watson-iot-table--with-toolbar
- Proof read docs and verify code examples here https://deploy-preview-3366--carbon-addons-iot-react.netlify.app/?path=/docs/1-watson-iot-table--with-toolbar
- Checkout main story here https://deploy-preview-3366--carbon-addons-iot-react.netlify.app/?path=/docs/1-watson-iot-table--playground and verify data download and row actions

**Regression Test (how to make sure this PR doesn't break old functionality)**
n.a



**Things to look for during review**
